eng/targets/Imports.targets

```diff
@@ -395,5 +395,8 @@
   <!-- Workaround for an issue where nested targets (e.g. GetCopyToPublishDirectoryItems in the SDK) call _SplitProjectReferencesByFileExistence without calling implict prerequisite AssignProjectConfiguration -->
   <Target Name="Workaround_SplitProjectReferencesByFileExistence" BeforeTargets="_SplitProjectReferencesByFileExistence" DependsOnTargets="AssignProjectConfiguration" />
 
+  <!-- Workaround for an issue where nested targets (e.g. CopyFilesToPublishDirectory in the SDK) call _CopyResolvedFilesToPublishPreserveNewest without calling implict prerequisite GenerateManifests -->
```

Comment:
Is there any link for this issue?

Reply:
fwiw, I reported similar msbuild ordering issues and it sounds like it's hard to fix: https://github.com/dotnet/msbuild/issues/10306

> adding a constraints to target order has been a surprisingly breaking operation in the past due to how people have hooked the fragile existing system.